addons/storysource/src/register.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { addons } from '@storybook/addons';
 
 import { StoryPanel } from './StoryPanel';
-import { ADDON_ID, PANEL_ID } from '.';
```

Comment:
This just makes it easier to actually find references to this file, the `import X from '.'` could resolve into many different things in some environments.

In fact, during my prebundling work this was actually a problem, so let's avoid it shall we?